### PR TITLE
Added import of tasks in the schedule manager

### PR DESCRIPTION
**Pull request by dkliban**

### CODE REVIEW

File: server/pulp/server/managers/schedule/repo.py [hunk 1]
```diff
@@ -12,6 +12,7 @@
 # http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
```
Comment (ipanova) [inline server/pulp/server/managers/schedule/repo.py]: can you also remove the copyright header?

**Comment by ipanova** [comment]
@dkliban  can you explain please why you added this import?

**Comment by ipanova** [comment]
ok test

**Comment by dkliban** [comment]
ok test

**Comment by dkliban** [comment]
ok test

**Comment by dkliban** [comment]
ok test

**Comment by bmbouter** [comment]
This makes sense to me.

**Pull request merged**